### ISSUE TITLE
Implement Thumbnail file type

### ISSUE THREAD

**Issue by awead** [projecthydra-labs/hydra-works#73]
Relates to projecthydra/sufia#1010

Build a thumbnail file class in HydraWorks such that an implementation could do:
``` ruby
class MyGenericFile < Hydra::Works::GenericFile
  has_one :thumbnail, predicate: ::RDF::URI(http://hydraworks.namespace/hasThumbnail), class_name: "HydraWorks::Thumbnail"
end
```
and triples associated with MyGenericFile would include `<> hasThumbnail HydraWorks::Thumbnail`

The thumbnail class extends Hydra::PCDM::File and has its own rdf type as per https://github.com/duraspace/pcdm/pull/5

**Comment by escowles** [comment]
See https://github.com/duraspace/pcdm/pull/7 -- adding a file use vocabulary as a PCDM extension, with RDF types and predicates for the identified file use terms.

**Comment by jcoyne** [comment]
I'm not sure why we'd need a class for this. Wouldn't a URI suffice?

**Comment by awead** [comment]
@jcoyne I could see that. I was just putting up an example to facilitate discussion.

**Comment by escowles** [comment]
This should probably be an AF::File with rdf:type=pcdmuse:ThumbnailImage added to the RDF description like in https://github.com/awead/sufia-pcdm/blob/master/pcdm-thumbnail.ru

**Comment by awead** [comment]
@escowles heh. just did that. see #79. I haven't added a property to the Work that points directly to the file, instead, it's just querying all the hasFile members and returning those with the pcdm.Thumbnail type.

**Comment by awead** [comment]
fixed in #105
